app/safe/internal/server/route/fetch/fetch.go

```diff
@@ -61,12 +62,12 @@ func Fetch(cid string, w http.ResponseWriter, r *http.Request, spiffeid string)
 
 	log.DebugLn(&cid, "Fetch: sending response")
 
-	defer func() {
-		err := r.Body.Close()
+	defer func(b io.ReadCloser) {
```

Comment:
moved the objects that we are interested to close into the closer as arguments to follow the pattern used in the majority of the codebase.